index.html

```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
+            "MUST" statements in Section <a href="#secure-with-sd-jwt"></a>.
+        </p>
+        <p>
+            A <dfn>conforming SD-JWT issuer implementation</dfn> produces
+            [=conforming SD-JWT documents=] and MUST secure them as described in Section
+            <a href="#secure-with-sd-jwt"></a>.
+        <p>
+            A <dfn>conforming SD-JWT verifier implementation</dfn> verifies
+            [=conforming SD-JWT documents=] as described in Section
+            <a href="#secure-with-sd-jwt"></a>.
+        </p>
+        <p>
+            A <dfn>conforming COSE document</dfn> is one that conforms to all of the
+            "MUST" statements in Section <a href="#secure-with-cose"></a>.
+        </p>
+        <p>
+            A <dfn>conforming COSE issuer implementation</dfn> produces
+            [=conforming COSE documents=] and MUST secure them as described in Section
+            <a href="#secure-with-cose"></a>.
+        </p>
+        <p>
+            A <dfn>conforming COSE verifier implementation</dfn> verifies
+            [=conforming COSE documents=] as described in Section
+            <a href="#secure-with-cose"></a>.
+        </p>
+    </section>
+    <section class="normative">
+        <h2 id="securing-verifiable-credentials">Securing Verifiable Credentials</h2>
+        <p>The <a data-cite="VC-DATA-MODEL-2.0#securing-mechanism-specifications"></a> describes
+            the approach taken by JSON Web Tokens to secure JWT Claims Sets as <i>applying an
+                <code>external proof</code></i>.
+        </p>
+        <p>The normative statements in <a data-cite="VC-DATA-MODEL-2.0#securing-mechanisms">Securing
+            Mechanisms</a> apply to securing
+            <code>application/vc-ld+jwt</code> and
+            <code>application/vp-ld+jwt</code>,
+            <code>application/vc-ld+sd-jwt</code> and
+            <code>application/vp-ld+sd-jwt</code>,
+            as well as
+            <code>application/vc-ld+cose</code> and
+            <code>application/vp-ld+cose</code>.
+        </p>
+        <p>
+            JSON Web Token implementers are advised to review <a data-cite="RFC7519#section-8">Implementation
+            Requirements</a>.
+        </p>
+        <p>
+            Accordingly, Issuers, Holders, and Verifiers MUST understand the
+            JSON Web Token header parameter
+            <code>"alg": "none"</code> when securing [[VC-DATA-MODEL-2.0]]
+            with JSON Web Tokens.
+            When content types from [[VC-DATA-MODEL-2.0]] are secured using
+            JSON Web Tokens, the header parameter <code>"alg": "none"</code>,
+            MUST be used to communicate that a JWT Claims Set (a
+            Verifiable Credential or a Verifiable Presentation) has no
+            integrity protection.
+            When a JWT Claims Set (a Verifiable Credential or a
+            Verifiable Presentation) contains
```

Comment:
This parenthetical note suggests that `a JWT Claims Set` could only be `a Verifiable Credential or a Verifiable Presentation`, but that's not so. If the intent is to say that these sentences only apply to any `JWT Claims Set` that *is* `a Verifiable Credential or a Verifiable Presentation`, they should be reworded, something like this ... and similar changes should be applied throughout the document.
```suggestion
            MUST be used to communicate that a JWT Claims Set that comprises a
            Verifiable Credential or a Verifiable Presentation has no
            integrity protection.
            When a JWT Claims Set that comprises a Verifiable Credential or a
            Verifiable Presentation contains
```